### DISCUSSION TITLE
Problem when import csv

### DISCUSSION THREAD

**Discussion by haidang2358** [dbeaver/dbeaver#11616] [Q&A]
I've tried to import many csv files, even 1 simple file with 2 columns Name and Age with just 2 records it still failed. Any idea?
<img width="838" alt="Capture5" src="https://user-images.githubusercontent.com/50066481/110062120-453d9a00-7d9b-11eb-87ba-20b2b160fab6.PNG">> 

<img width="945" alt="Capture6" src="https://user-images.githubusercontent.com/50066481/110062116-4373d680-7d9b-11eb-8c9d-71e77cbaeaa2.PNG">

**Comment by haidang2358** [comment]
Any help guys...

**Comment by LonwoLonwo** [comment]
Can you provide example your csv file here?
(May be it's delimiters trouble)
Also, do you import in an existing table or new?
And can I see you csv import setting? like on the picture 3 here https://github.com/dbeaver/dbeaver/wiki/Data-transfer#importing-data-from-csv-format

**Reply by haidang2358** [reply]
Hi,
-The same file but in csv, i changed format to xlsx to upload here
[Info.xlsx](https://github.com/dbeaver/dbeaver/files/6155600/Info.xlsx)
-I imported in new table
-here is the setting
![image](https://user-images.githubusercontent.com/50066481/111451838-711a3180-8744-11eb-8dee-c81728804e31.png)

**Comment by LonwoLonwo** [comment]
You can just copy content of csv file here. This is important, because I don't now what type of delimiter you use.

**Comment by LonwoLonwo** [comment]
With 
```
Name,age
Dang,26
Tu,23
```
And import in new MySQL table with same settings I have no problem.

**Comment by LonwoLonwo** [comment]
If you have the same CSV content (as I provided earlier), then please show me column mapping dialog

![2021-03-17 15_04_56-DBeaver Enterprise 21 0 0 - test_import_csv](https://user-images.githubusercontent.com/45152336/111464758-445d1e80-8732-11eb-8a71-a2613eb2cb40.png)

**Reply by haidang2358** [reply]
![image](https://user-images.githubusercontent.com/50066481/111466974-81d3a300-8756-11eb-8ae5-65574fea7e03.png)

**Comment by LonwoLonwo** [comment]
If you import in new table, why column "age" have "existing" status?

**Comment by LonwoLonwo** [comment]
O. Another idea.
If it is new table, can you show me "Target DDL" output?
![2021-03-17 15_43_28-DBeaver Enterprise 21 0 0 - _none_ Script-82](https://user-images.githubusercontent.com/45152336/111469337-a1a79e80-8737-11eb-975e-2cdf9821abca.png)

**Reply by haidang2358** [reply]
![image](https://user-images.githubusercontent.com/50066481/111469775-ad0bc180-8759-11eb-82f6-06e043ba314f.png)

**Reply by LonwoLonwo** [reply]
Gm. Unfortunately i don't have Hive base at the moment, can you try to execute this create table statement in SQL Editor? 
Also any error logs will be very helpful.
https://github.com/dbeaver/dbeaver/wiki/Log-files

**Reply by haidang2358** [reply]
It's ok when create table
![image](https://user-images.githubusercontent.com/50066481/111485684-6a51e580-8769-11eb-8295-f741034c803a.png)

**Comment by LonwoLonwo** [comment]
And that about error logs https://github.com/dbeaver/dbeaver/wiki/Log-files from original problem? "Error occurred during create target metadata"?
also do you can import in existing table?

**Reply by haidang2358** [reply]
This is the new created table after i imported, it failed as usual but when i read data in the editor the column "?name" was suspicious.
![image](https://user-images.githubusercontent.com/50066481/111663750-b07b7780-8843-11eb-81c4-5e84813b5d29.png)

**Reply by haidang2358** [reply]
I will try to update the error logs soon.

**Comment by LonwoLonwo** [comment]
And what if you try use another column name - not "Name"? Name1, name_copy - any other. 
What HIVE and DBeaver version do you use?
My HIVE db do not wrapped "Name" in quotes, unlike your.

**Reply by haidang2358** [reply]
after change column name to Name_1, still error
![image](https://user-images.githubusercontent.com/50066481/112001530-a2826b00-8b51-11eb-91e4-7995136ad94f.png)

![image](https://user-images.githubusercontent.com/50066481/112002326-584db980-8b52-11eb-9a6c-8fa947e80cc9.png)

some error log 
![image](https://user-images.githubusercontent.com/50066481/112005313-386bc500-8b55-11eb-8d43-8aa4a76a1b3f.png)

try to import to existing table still error

**Reply by LonwoLonwo** [reply]
Info from "Open log" https://github.com/dbeaver/dbeaver/wiki/Log-files#error-log-view - It would be nice to see

**Reply by LonwoLonwo** [reply]
And what about database version?
